packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1470,6 +1471,18 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.widgets.SelectionOverlay.onSelectionHandleTapped}
   final VoidCallback? onSelectionHandleTapped;
 
+  /// {@template flutter.widgets.editableText.groupId}
+  /// The group identifier for the [TextFieldTapRegion] of this text field.
+  ///
+  /// Text fields with the same group identifier share the same tap region.
+  /// Defaults to the type of [EditableText].
+  ///
+  /// See also:
+  ///
+  ///  * [TextFieldTapRegion], for the provision of a separate tap region.
```

Comment:
```suggestion
  ///  * [TextFieldTapRegion], to give a [groupId] to a widget that is to be included in a
  ///     [TextField]'s tap region that has [groupId] set.
```

Reply:
One nit on @gspencergoog's suggestion above: Maybe avoid saying `[TextField]` since this macro is reused in other classes. Probably say `[EditableText]`?

Reply:
thank you for your suggestions